### PR TITLE
feat(*): add definition list component (connects #8)

### PR DESCRIPTION
**Pull request by ephor**

### CODE REVIEW

**Comment by ivchenkodima** [comment]
Две правки. Переименуй please. И будем мерджить
Pencil -> PencilIcon
Profile -> ProfilePage

File: packages/components/src/SectionList.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React, { Fragment } from "react";
+
+const SectionList = ({ data, label, renderRow, renderHeader }) => (
+  <>
+    {renderHeader && renderHeader()}
```
Comment (kiurchv) [inline packages/components/src/SectionList.js]: I think that header is out of the scope of this component. Let's remove it from here and use separately on the page. BTW we have the `Title` component for this

File: packages/components/src/SectionList.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    {Object.entries(data).map(([key, text], index) => {
+      return <Fragment key={index}>{renderRow(text, label[key])}</Fragment>;
```
Comment (kiurchv) [inline packages/components/src/SectionList.js]: Let's use `<>` for fragments

File: packages/patient-account/src/components/DefinitionList.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      <DefinitionWrapper>
+        <Term>{label}</Term>
+        <Details>{detail}</Details>
+      </DefinitionWrapper>
+    )}
+    renderHeader={() =>
+      title && (
+        <SubTitle>
+          {title}
+          {onEdit && (
```
Comment (kiurchv) [inline packages/patient-account/src/components/DefinitionList.js]: Edit functionality is out of the scope of the DefinitionList component. My suggestion is to move all such functionality to the page component.

**Pull request merged**